domains/cert/no-sct/index.html

```diff
@@ -0,0 +1,16 @@
+---
+subdomain: no-sct
+layout: page
+favicon: red
+background: red
+---
+
+<div id="content">
+  <h1 style="font-size: 10vw;">
+    {{ page.subdomain }}.<br>{{ site.domain }}
+  </h1>
+</div>
+
+<div id="footer">
+  The server does not send a <a href="https://www.certificate-transparency.org/how-ct-works">Signed Certificate Timestamps (SCTs)</a> for this domain.
```

Comment:
```suggestion
  The server does not send <a href="https://www.certificate-transparency.org/how-ct-works">Signed Certificate Timestamps (SCTs)</a> for this domain.
```

nit: match singular/plural

Reply:
Hmmm, SCTs are currently plural in this sentence (since there can be plural SCTs). Either seems fine for me, but if you have a strong opinion could you update the text content of the link too?

Reply:
I went with plural in my suggestion, but I guess it's more accurate to be singular (you would only expect the server to send a single SCT, if embedded in the X509 extension or via the TLS extension, I think). Updated my suggestion to be "a Signed Certificate Timestamp (SCT)". WDYT? (I don't feel strongly either way, it's just a grammar nit.)

Reply:
Singular sounds good to me!